esi-custom.yaml

```diff
@@ -1,4 +1,29 @@
+resource_registry:
+  OS::TripleO::Services::NovaCompute: OS::Heat::None
+  OS::TripleO::Services::NovaLibvirt: OS::Heat::None
+  OS::TripleO::Services::Redis: /usr/share/openstack-tripleo-heat-templates/deployment/database/redis-pacemaker-puppet.yaml
+
 parameter_defaults:
+  NeutronBridgeMappings: datacentre:br-ctlplane
+  NeutronPhysicalBridge: br-ctlplane
+  NeutronFlatNetworks: datacentre
+  NeutronMechanismDrivers: [openvswitch, baremetal]
+  NeutronNetworkVLANRanges: datacentre:611:620
```

Comment:
I think we need to reserve this in MOC documents too. Is this the VLAN range that will be assigned to tenants? 
If yes, then we should definitely increase this.

Reply:
Yep! Let us know what we can set this to; for now I think it's fine for testing.

Reply:
Oh, again, I copied it from demo server. So I think this range is reserved for demo server?

Reply:
okay, if we are still testing then it's fine. I guess we should know if we can expand this after the deployment.
We can have 600 to 900.